starlette_admin/translations/tr/LC_MESSAGES/admin.po

```diff
@@ -157,10 +157,18 @@ msgstr "Hesabınıza giriş yapın"
 msgid "Username"
 msgstr "Kullanıcı adı"
 
+#: starlette_admin/templates/login.html:36
+msgid "Enter username"
+msgstr "Kullanıcı adı"
```

Comment:
Can you confirm if this is accurate?

Reply:
Yep, I checked some major Turkish websites. They are not saying "enter username" or "put username", just "username". For context, we should follow their approach.